src/cirrus/management/commands/manage.py

```diff
@@ -314,3 +314,126 @@ def get_payloads(
         rerun,
     ):
         click.echo(json.dumps(payload, default=str))
+
+
+@manage.command("list-workflows")
+@pass_deployment
+def list_workflows(deployment: Deployment):
+    """List available collections/workflows combinations"""
+    workflows = deployment.get_workflows()
+    click.echo(json.dumps(workflows, indent=2))
+
+
+@manage.command("get-workflow-summary")
+@click.argument("collections")
+@click.argument("workflow_name")
+@click.option(
+    "--since",
+    default=None,
+    help=(
+        "Only include items updated since this relative duration (e.g., 7d, 36h, 15m)"
+    ),
+)
+@click.option(
+    "--limit",
+    default=100000,
```

Comment:
Perhaps the default in the deployments class can remain `100000`, but in CLI land, it can be ratcheted down to something that doesn't flood the terminal, say 10 or 100?